Source: maxice8
Upstream: no
Reason: restores version of libGLESv1 library that was on autotools

--- src/mapi/es1api/meson.build
+++ src/mapi/es1api/meson.build
@@ -38,7 +38,7 @@ libglesv1_cm = shared_library(
   include_directories : [inc_src, inc_include, inc_mapi],
   link_with : libglapi,
   dependencies : [dep_thread, dep_libdrm, dep_m, dep_dl],
-  version : '1.0.0',
+  version : '1.1.0',
   install : true,
 )
 
